src/styles/sciml/nest.jl

```diff
@@ -38,7 +38,14 @@ function n_binaryopcall!(ss::SciMLStyle, fst::FST, s::State; indent::Int = -1)
     line_margin = s.line_offset + length(fst) + fst.extra_margin
     if line_margin > s.opts.margin &&
        fst.ref !== nothing &&
-       CSTParser.defines_function(fst.ref[])
+       (
```

Comment:
@ChrisRackauckas should another check for `short_to_long_function_def` be added here or is it intentional to not check?

Reply:
the check is not in the pass itself so all that's left is verify that we can to make try transform always, even if `short_to_long_function_def` is set to `false`

Reply:
> should another check for short_to_long_function_def be added here or is it intentional to not check?

That I don't know, @YingboMa wrote that I think?